README.md

```diff
@@ -1,59 +1,50 @@
-# Getting Started with Create React App
+# clockwork
+[![Contributors][contributors-shield]][contributors-link]
+[![Stargazers][stars-shield]][stars-link]
+[![Issues][issues-shield]][issues-link]
+[![Website][website-shield]][website-link]
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Clockwork is a website built to simplify the experience of searching courses and building timetables for the [University of Victoria](https://uvic.ca) (UVic). The website is a TypeScript app which uses React on the front-end and Firebase on the back-end.
```

Comment:
I'm fine with either aesthetic, you can change it back if you want. but other than that, lgtm.